tests/landscape.sil

```diff
@@ -0,0 +1,4 @@
+\begin[orientation=landscape,papersize=a5]{document}
+\use[module=packages.lorem]
+\lorem[words=1000]
```

Comment:
Lets not use lorem for this if we don't need to since we don't need a lot of content to prove that the papersize got changed, the first lie of debug output is enough. Everything else is noise in the expectation file that can change and make huge diffs to review when unrelated factors change. I think the MWE for this test can be a completely empty document, the debug output shows the canvas measurements and they reflect the landscape orientation.